vspace/src/main/java/edu/asu/diging/vspace/core/data/ImageRepository.java

```diff
@@ -14,15 +13,19 @@
 
 @Repository
 @JaversSpringDataAuditable
-public interface ImageRepository extends PagingAndSortingRepository<VSImage, String> {
+public interface ImageRepository extends PagingAndSortingRepository < VSImage, String > {
+
+    List < VSImage > findByFilenameLikeOrNameLike(String filename, String name);
+
+    List < VSImage > findByFilenameLikeOrNameLikeOrDescriptionLike(String filename, String name, String description);
+
+    Page < VSImage > findByFilenameLikeOrNameLikeOrDescriptionLike(Pageable pageable, String filename, String name, String description);
 
-    List<VSImage> findByFilenameLikeOrNameLike(String filename, String name);
+    List < VSImage > findByFilenameLike(String filename);
 
-    List<VSImage> findByFilenameLike(String filename);
+    Page < VSImage > findByCategories(Pageable pageable, ImageCategory category);
 
-    Page<VSImage> findByCategories(Pageable pageable, ImageCategory category);
+    List < IVSImage > findByCategories(ImageCategory category);
```

Comment:
the shouldn't be blanks after/before `<` and `>`

Reply:
Removed blanks from <>.